projects/kit/utils/miscellaneous/replay-control-value-changes-factory.ts

```diff
@@ -12,10 +11,8 @@ import {map} from 'rxjs/operators';
 
 /**
  * @internal
- * @deprecated use {@link tuiReplayControlValueChangesFactory}
- * TODO: 3.0 name should be shorter and start with lowercase letter.
  */
-export function TuiReplayControlValueChangesFactory<
+export function tuiReplayControlValueChangesFactory<
```

Comment:
We also wanted to make the name shorter :)

Reply:
renamed to `tuiReplayedValueChangesFactory`, don't know how to make it shorter
anyway function is internal, can be renamed in any moment

Reply:
I'd just call it `tuiControlValueFactory` and `tuiReplayedValueChangesFrom` to `tuiControlValue`. JSDocs and return type should make it clear what these functions do.